### ISSUE TITLE
SPG Deployment Script Issue

### ISSUE THREAD

**Issue by bpolania** [storyprotocol/protocol-periphery-v1#22]
## Description and context

An issue in the deployment script results in an `OwnableUnauthorizedAccount` error when run directly. This error stems from the manner in which the `UpgradeableBeacon` is instantiated. 

## Solution recommendation

57Blocks team suggest the following modification to the deployment script:

```
//Before:
abi.encodePacked(type(UpgradeableBeacon).creationCode, abi.encode(address(spgNftImpl), deployer))

//After:
abi.encodePacked(type(UpgradeableBeacon).creationCode, abi.encode(address(spgNftImpl), address(this)))
```

**Comment by sebsadface** [comment]
This issue was caused by the incorrect ordering of the `_endBroadcast();` call and the `transferOwnership(address(spg));` call in the deployment script.

**This issue should already be fixed by #18.**

Please pull the latest deployment code and try again. Let me know if the issue persists.

**Comment by bpolania** [comment]
This issue was already fixed in this PR https://github.com/storyprotocol/protocol-periphery-v1/issues/22
